### ISSUE TITLE
NC region doesn't repaint when dragging in MDIParent

### ISSUE THREAD

**Issue by ukandrewc** [MicrosoftEdge/WebView2Feedback#1257]
**Description**
MDI Child doesn't repaint NC region when dragging

**Version**
SDK: 1.0.865.0 prerelease

![image](https://user-images.githubusercontent.com/30246320/117256338-79d4dd00-ae42-11eb-99d3-90b8218150c9.png)

**Comment by champnic** [comment]
Thanks for the bug report @ukandrewc! To confirm, the error in the image above is the border around the foreground "Form1" window is not redrawing and so is showing errors rather than light blue?

**Comment by ukandrewc** [comment]
@champnic Yes, the border and the shadow don't refresh when dragging. This is only when the form is an MDIChild. As two standalone forms, eveything is good.
